coriolisclient/v1/logging.py

```diff
@@ -22,6 +22,7 @@
 import websockets
 
 from keystoneauth1.exceptions.catalog import EndpointNotFound
+from keystoneauth1.exceptions.http import Unauthorized
```

Comment:
We should always avoid direct imports (i.e. `from module_name import exact_function_name`) to avoid the risk of name shadowing (i.e. there being another `Unauthorized` object/thing in the same namespace)

By just doing `from keystoneauth1.exceptions import catalog; catalog.EndpointNotFound`, we can avoid any risk whatsoever.

Could you please do so for all of the imports in this file?